Dockerfile

```diff
@@ -15,7 +15,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-ARG GO_VERSION=1.21.4
```

Comment:
curious if you ran into issues with 1.21.4, because there was a regression on Windows in that version (related to previous security fixes)